visualization/views/total_contracts_view.py

```diff
@@ -34,23 +31,14 @@ def get(self, request):
         if supplier:
             filter_args = add_filter_args("supplier", supplier, filter_args)
 
-        total_country_tender = Tender.objects.filter(**filter_args).count()
         bar_chart = (
             Tender.objects.filter(**filter_args)
             .values("procurement_procedure")
-            .annotate(count=Count("procurement_procedure"))
+            .annotate(count=Count("procurement_procedure"), total_country_count=Count("id"))
```

Comment:
rename `total_country_count` to `total_contract` in `total_country_count=Count("id")`